tests/observability/metrics/test_vms_metrics.py

```
@@ -509,3 +510,22 @@ def test_kubevirt_vmi_phase_transition_from_deletion_seconds_sum_windows(
+class TestVmCreatedByPodTotal:
+    @pytest.mark.parametrize(
+        "initial_metric_value_scope_module",
+        [
+            pytest.param(
+                f"sum({KUBEVIRT_VM_CREATED_BY_POD_TOTAL})",
+                marks=pytest.mark.polarion("CNV-12361"),
+            )
+        ],
+        indirect=True,
+    )
+    def test_kubevirt_vm_created_by_pod_total(self, prometheus, initial_metric_value_scope_module, vm_list):
```

Ruff (0.14.0)

Unused method argument: `vm_list`

(ARG002)